src/index.html

```diff
@@ -5,4 +5,151 @@
 order: 1
 ---
 
-<p>Add the HTML for your homepage here.</p>
+<iframe src="https://www.youtube.com/embed/z_jefhcVRAk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<div class="about">
+    <div class="img">
+        <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fabout.png?v=1605143407347" alt="Dancing Girl">
+    </div>
+    <div class="text">
+        <h1>About MSDC</h1>
+        <p>Originally established in 1992, Morton Street Dance Center is one of Baltimore's best and most comprehensive dance schools.</p>
+        <button class="more">Learn More</button>
+    </div>
+</div>
+<div class="offerings">
+        <h1>Our CLass Offerings</h1>
+    <ul>
+        <li class="ballet">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fballet.png?v=1605143408910" alt="Ballet">
```

Comment:
Here, the `alt` attribute just repeats the heading that follows, which is not ideal -- this means someone using a screen reader hears "image, ballet, heading, ballet" and the image `alt` attribute isn't providing them any additional information.

Two things to consider:
1. Using a blank `alt` attribute like so: `alt=""`
2. Writing more descriptive `alt` values that convey some detail about the dancers pictured. This would give screen reader users a more equal experience.

This same comment applies to all the `img` elements in this list, but I won't repeat it for each.